sdk/keyvault/keyvault-keys/src/index.ts

```diff
@@ -111,7 +112,8 @@ export {
   EncryptionAlgorithm,
   KeyOperation,
   KeyType,
-  KeyPollerOptions,
```

Comment:
Still need to export KeyPollerOptions since its a base interface for exported types.

Reply:
Done here: https://github.com/Azure/azure-sdk-for-js/pull/5929/commits/dcb7f79a057cf7a61bf50b9625e995c62fd7a98d